tests/test_message.py

```diff
@@ -482,6 +482,46 @@ async def make_assertion(*args, **kwargs):
         if reply_parameters is None or reply_parameters.message_id != 42:
             pytest.fail(f"reply_parameters is {reply_parameters} but should be 42")
 
+    @staticmethod
+    async def check_thread_id_parsing(
+        message: Message, method, bot_method_name: str, args, monkeypatch
+    ):
+        """Used in testing reply_* below. Makes sure that meassage_thread_id is parsed
+        correctly."""
+
+        async def make_assertion(*args, **kwargs):
```

Comment:
```suggestion
        async def extract_message_thread_id(*args, **kwargs):
```
Didn't really assert anything:)